client/src/components/App/App.js

```diff
@@ -1,21 +1,86 @@
 import React, { Component } from 'react';
 import './App.css';
 import styled from 'styled-components';
-import NavBar from '../NavBar';
+let password = '',
+    confirmPassword ='';
 
 
 class App extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      signUpEmail: '',
+      signUpPassword:'',
+      signUpPasswordConfirm:''
+    }
+  }
+
   render() {
     return (
       <div>     
           Sign Up <br/>
-          <input
-              type="text"
-              placeholder="First Name"/><br/> 
-          <input type="submit" value="Submit" />
+          <form>
+            <input
+              type="email"
+              name="signUpEmail"
+              placeholder="Email"
+              value={this.state.signUpEmail}
+              onChange={this.handleChange}/><br/> 
+             <input
+              type="password"
+              name="signUpPassword"
+              placeholder="Password"
+              value={this.state.signUpPassword}
+              onChange={this.handlePasswordChange}/><br/>
+            <input
+              type="password"
+              name="signUpPasswordConfirm"
+              placeholder="Confirm Password"
+              value={this.state.signUpPasswordConfirm}
+              onChange={this.handleConfirmPasswordChange}/><br/>  
+            <input 
+              type="submit" 
+              value="Submit" 
+              onClick={this.validateInput}/>
+          </form> 
       </div>
     );
   }
-}
 
+  handleChange = (event) => {
+    this.setState({[event.target.name]: event.target.value});   
+  }
+  handlePasswordChange = (event) => {
+    this.handleChange(event);
+    password = event.target.value;
```

Comment:
@amtadros Using global variables like this is usually a bad habit, you should be able to access it in state where you need it.

Reply:
@blaketarter Ok, got the global variables removed! Let me know you see anything else I need to update before this PR can be merged :)